lib/addons/neuron/neuron-monitor.yaml

```diff
@@ -0,0 +1,85 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: neuron-monitor
+  namespace: kube-system
+  labels:
+    app: neuron-monitor
+    role: master
+spec:
+  selector:
+    matchLabels:
+      app: neuron-monitor
+      role: master
+  template:
+    metadata:
+      labels:
+        app: neuron-monitor
+        role: master
+    spec:
+      containers:
+      - name: app
+        image: public.ecr.aws/h6c7e9p3/neuron/neuron-tools:latest
```

Comment:
should it be hard-coded to latest always? do you think customers will not want to set the version explicitly for this image?

Reply:
Adressed